src/WebpackChunkModuleIterator.ts

```diff
@@ -9,18 +9,19 @@ class WebpackChunkModuleIterator {
   iterateModules(
     compilation: WebpackCompilation,
     chunk: WebpackChunk,
+    stats: WebpackStats | undefined,
     callback: ((module: WebpackChunkModule) => void)
   ): void {
-    if (typeof compilation.chunkGraph !== 'undefined') {
+    if (
+      typeof compilation.chunkGraph !== 'undefined' &&
+      typeof stats !== 'undefined'
```

Comment:
Any one of these checks is enough to ensure we are running in WP5, but we include both mainly for type-narrowing (so typescript knows chunkGraph and stats are defined in this case)